web/src/tippyjs.ts

```diff
@@ -502,10 +503,24 @@ export function initialize(): void {
     });
 
     delegate("body", {
-        target: "#user_info_popover .status-emoji",
+        target: ".user-card-status-text .status-emoji",
         appendTo: () => document.body,
     });
```

Comment:
What's this change do?

Reply:
OK -- can you put it in its own commit (either a prep commit or new final commit)? I think a commit just after that one renaming that block from `user-card-status-text` to `user-card-status-area` would be a big help; the current name really hints (incorrectly) that it's just for a textual status, and not emoji.

Check out our [GitHub guide](https://zulip.readthedocs.io/en/latest/git-guide.html) and [commit guidelines](https://zulip.readthedocs.io/en/latest/contributing/commit-discipline.html) for details on why this is important and really helpful.

Reply:
Sure thing. Even I found it a little vague to add the `user-card-status-text` class but I was unsure.
Updated the changes in different commits.